ParProcCo/nxdata_aggregator.py

```diff
@@ -222,7 +226,7 @@ def _write_aggregation_file(self, aggregation_output: Path, output_data_files: L
             process = processed.create_group("process")
             process.attrs["NX_class"] = "NXprocess"
             process.create_dataset("date", data=str(datetime.now(timezone.utc)))
-            process.create_dataset("parameters", data=f"inputs: {output_data_files}, output: {aggregation_output}")
+            process.create_dataset("parameters", data=f"inputs: {self.data_files}, output: {aggregation_output}")
```

Comment:
Can remove `output_data_files` from `_write_aggregation_file` args on L219 and L45 and replace with `self.data_files` on L250